webui/react/src/e2e/models/components/CreateUserModal.ts

```diff
@@ -25,6 +25,10 @@ export class CreateUserModal extends Modal {
     parent: this.body,
     selector: '[data-testid="isRemote"] button',
   });
+  readonly password: BaseComponent = new BaseComponent({
```

Comment:
can you include the form area so we can get PASSWORD_RULES and check for those appearing properly? it could be in another PR if you want